ide/defaults/src/org/netbeans/modules/defaults/VSCode-keybindings.xml

```diff
@@ -0,0 +1,112 @@
+    <bind actionName="move-selection-else-line-down" key="A-DOWN"/>
+    <bind actionName="move-selection-else-line-up" key="A-UP"/>
+    <bind actionName="copy-selection-else-line-down" key="SA-DOWN"/>
+    <bind actionName="copy-selection-else-line-up" key="SA-UP"/>
+    <bind actionName="remove-line" key="CS-K"/>
+    <bind actionName="start-new-line" key="C-ENTER"/>
+    <bind actionName="insert-break-above" key="CS-ENTER"/>
+    <bind actionName="match-brace" key="CS-BACK_SLASH"/>
+    <bind actionName="indent" key="C-CLOSE_BRACKET"/>
+    <bind actionName="unindent" key="C-OPEN_BRACKET"/>
+    <bind actionName="caret-begin-line" key="HOME"/>
+    <bind actionName="caret-end-line" key="END"/>
+    <bind actionName="caret-begin" key="C-HOME"/>
+    <bind actionName="caret-end" key="C-END"/>
+    <bind actionName="scroll-up" key="C-UP"/>
+    <bind actionName="scroll-down" key="C-DOWN"/>
+    <bind actionName="page-up" key="A-PAGE_UP"/>
+    <bind actionName="page-down" key="A-PAGE_DOWN"/>
+    
+    <!-- Code folding -->
+    <bind actionName="collapse-fold" key="CS-OPEN_BRACKET"/>
+    <bind actionName="expand-fold" key="CS-CLOSE_BRACKET"/>
+    <bind actionName="collapse-all-folds" key="C-K C-OPEN_BRACKET"/>
+    <bind actionName="expand-all-folds" key="C-K C-CLOSE_BRACKET"/>
+    
+    <!-- Comments -->
+    <bind actionName="toggle-comment" key="C-SLASH"/>
+    <bind actionName="toggle-comment" key="C-K C-C"/>
+    <bind actionName="remove-comment" key="C-K C-U"/>
+    <bind actionName="toggle-block-comment" key="SA-A"/>
+    
+    <!-- Search and navigation -->
+    <bind actionName="find" key="C-F"/>
+    <bind actionName="replace" key="C-H"/>
+    <bind actionName="find-next" key="F3"/>
+    <bind actionName="find-previous" key="S-F3"/>
+    <bind actionName="find-selection" key="C-F3"/>
+    <bind actionName="goto" key="C-G"/>
+    <bind actionName="goto-declaration" key="F12"/>
+    <bind actionName="goto-source" key="A-F12"/>
+    <bind actionName="goto-implementation" key="C-F12"/>
+    
+    <!-- Multi-cursor and selection -->
+    <bind actionName="select-word" key="C-D"/>
+    <bind actionName="select-next-parameter" key="C-L"/>
+    <bind actionName="select-all" key="C-A"/>
+    <bind actionName="selection-grow" key="SA-RIGHT"/>
+    <bind actionName="selection-shrink" key="SA-LEFT"/>
+    
+    <!-- Editor management -->
+    <bind actionName="close" key="C-F4"/>
+    <bind actionName="close" key="C-W"/>
+    <bind actionName="split-editor" key="C-BACK_SLASH"/>
+    
+    <!-- Rich language features -->
+    <bind actionName="completion-show" key="C-SPACE"/>
+    <bind actionName="completion-show" key="C-I"/>
+    <bind actionName="parameter-hints-show" key="CS-SPACE"/>
+    <bind actionName="format" key="SA-F"/>
+    <bind actionName="fix-imports" key="C-DOT"/>
+    
+    <!-- Refactoring -->
+    <bind actionName="rename" key="F2"/>
+    <bind actionName="refactor-quickmenu" key="C-DOT"/>
```

Comment:
`C-DOT` doesn't appear to be a valid keystroke, what would work is `C-PERIOD`

also: the same keystroke is used for two refactoring actions which is not ideal